homeassistant/components/home_connect/sensor.py

```diff
@@ -179,6 +179,11 @@ class HomeConnectSensorEntityDescription(
         ],
         translation_key="last_selected_map",
     ),
+    HomeConnectSensorEntityDescription(
+        key=StatusKey.COOKING_OVEN_CURRENT_CAVITY_TEMPERATURE,
+        device_class=SensorDeviceClass.TEMPERATURE,
```

Comment:
can we also set the `state_class` to `MEASUREMENT`?

Reply:
Sure!